### PR TITLE
Improve the template file on new test file.

### PR DESCRIPTION
**Pull request by RoyalMist**
Hello I was thinking about adding this to the base template of a new file like so:

defmodule Hangman.RealTest do
	use ExUnit.Case
	doctest Hangman.Real
	alias Hangman.Real

end

### CODE REVIEW

File: src/commands/jumpToTest.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ async function createNewTestFile(dir, file) {
   ws.insert(
     uriFile,
     new vscode.Position(0, 0),
-    `defmodule ${moduleName}Test do\nend`,
+    `defmodule ${moduleName}Test do\n\tuse ExUnit.Case\n\tdoctest ${moduleName}\n\talias ${moduleName}\n\nend`,
```
Comment (samuelpordeus) [inline src/commands/jumpToTest.js:35]: Can you add a confirmation dialog here to see if the user wants to add the doctest line?

https://github.com/samuelpordeus/vscode-elixir-test/blob/fef6849a14e9b02e5bb3f434717ea4dca25ca01d/src/commands/jumpToTest.js#L42

Comment (RoyalMist) [reply src/commands/jumpToTest.js:35]: Ok I will lookt at it :)

Comment (RoyalMist) [reply src/commands/jumpToTest.js]: @samuelpordeus here it is :)

**Comment by RoyalMist** [comment]
Hello @samuelpordeus so there are now two dialogs in a row hope this is ok for you,

**Comment by samuelpordeus** [comment]
@RoyalMist mmm... what is your opinion about it? Does it look bad?

**Comment by RoyalMist** [comment]
> @RoyalMist mmm... what is your opinion about it? Does it look bad?

I propose to see it while using the feature. At least it can save time so I think it is ok :)

**Comment by ricardoruwer** [comment]
I'd say that 1 dialog is better 😄

**Comment by RoyalMist** [comment]
> I'd say that 1 dialog is better 😄

Yes :)
Maybe adding doctest by default is not harmfull since if the user did not add Doctest in the module it will have no effect wht do you think ?

**Comment by samuelpordeus** [comment]
@RoyalMist :shipit: 👍

**Comment by samuelpordeus** [comment]
@RoyalMist Thanks for the initiative and contribution! ❤️

**Pull request merged**